### PR TITLE
Remove not relevant and duplicate content

### PR DESCRIPTION
**Pull request by 75asu**
This commit removes the duplicate link which was provided at the beginning of the page, rectifies few grammatical errors.

Signed-off-by: Asutosh Panda <78539272+measutosh@users.noreply.github.com>

**Description**

This PR removes not relevant and duplicate content, fixes few grammatical erros.

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7f5390ef3f1c0bf1d5e5405de3df29da2ebe98d4 at: https://631335c36702a701dc89dde8--meshery-docs-preview.netlify.app

**Review by legendarykamal — CHANGES_REQUESTED** [review]
please redirect to this url https://docs.meshery.io/extensibility/adapters

**Comment by measutosh** [comment]
> please redirect to this url https://docs.meshery.io/extensibility/adapters

Made the relevant suggested changes, please have a look

**Comment by l5io** [comment]
🚀 Preview for commit 2ff64e09ab6d3819d0f24cff519c2901bf740850 at: https://6313b97c1b29b15a62b253aa--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-adapters.md
Comment (legendarykamal) [inline docs/pages/project/contributing/contributing-adapters.md:11]: Add this complete -> Extensibility: Service Mesh Adapters

**Review by legendarykamal — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/contributing-adapters.md
Comment (measutosh) [reply docs/pages/project/contributing/contributing-adapters.md:11]: Understood but could you please suggest why do we need to add the "see extensibility service mesh" at the initial.
Anyhow we are using it below at the end of the line right.
So I felt that it got repeated twice which is not needed.
Correct if i am wrong somewhere in this

Comment (legendarykamal) [reply docs/pages/project/contributing/contributing-adapters.md:11]: yes good point @measutosh  , remove that See line , we dont need that 
I had asked for the bottom line after 11 add Adapters Word

Comment (measutosh) [reply docs/pages/project/contributing/contributing-adapters.md]: Understood, made the suggested change

And i have given few comments on the other PR I raised which you closed, could you please check there once. 

Thanks

**Review by legendarykamal — APPROVED** [review]
LGTM :balloon:

**Comment by l5io** [comment]
🚀 Preview for commit cbc9409b380450b99ddcb2e0c2ec12208d05b26b at: https://6313c22580fe055e018f1df8--meshery-docs-preview.netlify.app

**Comment by measutosh** [comment]
> LGTM 🎈

Thanks Kamal
It was my 1st PR✌️

**Comment by measutosh** [comment]
> LGTM 🎈

Hi @legendarykamal , the PR is okay as per you right, any other issues with the PR? or there are some rules to merge the PR like in tuesday or wednesday? anything like such?

**Comment by legendarykamal** [comment]
@measutosh have little patience, I have approved it, a maintainer will soon check this and merge it . 
And ya no rules like that for merging days, as soon as a maintainer feels it's good to merge, it gets 👍

File: docs/pages/project/contributing/contributing-adapters.md
Comment (warunicorn19) [inline docs/pages/project/contributing/contributing-adapters.md:11]:
```suggestion
Meshery Adapters are the Extension Points in Meshery's architecture. Their design, the process of creating a new adapter and guidelines for contributing to an existing adapter is documented in [Extensibility: Service Mesh Adapters]({{site.baseurl}}/extensibility/adapters).
```

**Review by warunicorn19 — APPROVED** [review]
LGTM. Thanks @measutosh

**Comment by legendarykamal** [comment]
> > LGTM balloon
> 
> Thanks Kamal It was my 1st PRv

Yay  @measutosh  got merged :confetti_ball:

**Pull request merged**
